docs/source/core/exp_manager.rst

```diff
@@ -73,7 +73,7 @@ via YAML or CLI:
 Experiment Loggers
 ------------------
 
-Alongside Tensorboard, NeMo also supports Weights and Biases, MLFlow and DLLogger. To use these loggers, simply set the following
+Alongside Tensorboard, NeMo also supports Weights and Biases, MLFlow, DLLogger, ClearML and NeptuneLogger. To use these loggers, simply set the following
```

Comment:
@titu1994  I am closing this PR as I am not able to ammend a particular commit to add SignOff. I will reopen a new PR with the consolidated changes. Hope that is fine.

Reply:
Closing this PR in favor of [this](https://github.com/NVIDIA/NeMo/pull/8210) PR.